src/resolv/stub/conf.rs

```diff
@@ -452,10 +452,16 @@ impl ResolvConf {
         words: SplitWhitespace
     ) -> Result<(), Error> {
         // XXX TODO
-        let _ = words; 
-        Ok(())
+        let _ = words;
+        let ret = Ok(());
+
+        if ret.is_err() {
```

Comment:
Maybe it is better to comment out `parse_sortlist` entirely and replace its call in the match with a noop.

Reply:
My preference would be to replace the match or the fn body with `todo!()` but I'll comment it out.